src/Spryker/Client/RabbitMq/Model/Connection/ConnectionManager.php

```diff
@@ -8,201 +8,89 @@
+use PhpAmqpLib\Channel\AMQPChannel;
 use Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigFilter\ConnectionConfigFilterInterface;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigMapper\ConnectionConfigMapperInterface;
+use Spryker\Client\RabbitMq\Model\Connection\ConnectionCreator\ConnectionCreatorInterface;
 use Spryker\Client\RabbitMq\Model\Exception\DefaultConnectionNotFoundException;
+use Spryker\Client\RabbitMq\RabbitMqConfig;
 
 class ConnectionManager implements ConnectionManagerInterface
 {
     /**
-     * @var \Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface
+     * @var \Spryker\Client\RabbitMq\RabbitMqConfig
      */
-    protected $storeClient;
+    protected $config;
 
     /**
-     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionFactoryInterface
-     */
-    protected $connectionFactory;
-
-    /**
-     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionInterface[]|null Keys are connection names.
-     */
-    protected $connectionMap;
-
-    /**
-     * @var array|null Keys are pool names, values are lists of connections.
+     * @var \Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface
      */
-    protected $connectionMapByPoolName;
+    protected $storeClient;
 
     /**
-     * @var array|null Keys are store names, values are lists of connections.
+     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigMapper\ConnectionConfigMapperInterface
      */
-    protected $connectionMapByStoreName;
+    protected $connectionConfigMapper;
 
     /**
-     * @var string|null
+     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigFilter\ConnectionConfigFilterInterface
      */
-    protected $defaultConnectionName;
+    protected $connectionConfigFilter;
 
     /**
-     * @var array|null
+     * @var \Spryker\Client\RabbitMq\Model\Connection\ConnectionCreator\ConnectionCreatorInterface
      */
-    protected $connectionNameLocaleMap;
+    protected $connectionCreator;
 
     /**
+     * @param \Spryker\Client\RabbitMq\RabbitMqConfig $config
      * @param \Spryker\Client\RabbitMq\Dependency\Client\RabbitMqToStoreClientInterface $storeClient
-     * @param \Spryker\Client\RabbitMq\Model\Connection\ConnectionFactoryInterface $connectionFactory
-     */
-    public function __construct(RabbitMqToStoreClientInterface $storeClient, ConnectionFactoryInterface $connectionFactory)
-    {
+     * @param \Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigMapper\ConnectionConfigMapperInterface $connectionConfigMapper
+     * @param \Spryker\Client\RabbitMq\Model\Connection\ConnectionConfigFilter\ConnectionConfigFilterInterface $connectionConfigFilter
+     * @param \Spryker\Client\RabbitMq\Model\Connection\ConnectionCreator\ConnectionCreatorInterface $connectionCreator
+     */
+    public function __construct(
```

Comment:
In changelogs you have row `Introduced ConnectionManager `, but it's not a new model. You should describe what changed in this model. Also please check other changelogs